tests/integration/targets/synchronize/tasks/main.yml

```
@@ -227,3 +227,36 @@
   - directory_a/foo.txt
   - directory_a
   - directory_b
+- name: synchronize files with quiet option
+  synchronize:
+    src: '{{output_dir}}/foo.txt'
+    dest: '{{output_dir}}/foo.result'
+    quiet: true 
+  register: sync_result
+  ignore_errors: true
+- assert:
+    that:
+    - '''files/directories have been synchronized'' in sync_result.msg'
+- name: Cleanup
+  file:
+    state: absent
+    path: '{{output_dir}}/{{item}}'
+  with_items:
```

this begins to become more than deprecated...
```suggestion
  loop:
```